tests/platform_tests/api/test_fan_drawer_fans.py

```diff
@@ -34,13 +34,15 @@
 
 STATUS_LED_COLOR_GREEN = "green"
 STATUS_LED_COLOR_AMBER = "amber"
+STATUS_LED_COLOR_ORANGE = "orange"
```

Comment:
Why do we need both `amber` and `orange`? They are essentially the same color.

Reply:
Fixed.